src/Factory.sol

```diff
@@ -247,23 +269,23 @@ contract Factory {
                 timelock,
                 abi.encodeCall(
                     WithdrawalQueue.initialize,
-                    (timelock, vaultConfig.nodeOperator) // (admin, finalizerRoleHolder)
+                    (timelock, _vaultConfig.nodeOperator) // (admin, finalizerRoleHolder)
                 )
             )
         );
 
-        address distributor = DISTRIBUTOR_FACTORY.deploy(vaultConfig.nodeOperator, vaultConfig.nodeOperatorManager);
+        address distributor = DISTRIBUTOR_FACTORY.deploy(_vaultConfig.nodeOperator, _vaultConfig.nodeOperatorManager);
```

Comment:
nodeOperatorManager is responsible for rewards distribution here, although nodeOperator is responsible for finalisation. looks like both are some operational work and could be assign to the same address. consider adding a posibility to configure this address or set it to the same as for finalisation